po/fr.po

```diff
@@ -277,18 +276,20 @@ msgid "invalid feed index (bug)"
 msgstr "Index de fil invalide (bug)"
 
 #: src/controller.cpp:1164
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "Newsboat is free software licensed under the MIT License. (Type `%s -vv' to "
 "see the full text.)"
 msgstr ""
-"newsboat est un logiciel libre, publié sous la licence MIT/X Consortium."
+"newsboat est un logiciel libre, publié sous la licence MIT/X Consortium. (Tapez `%s -vv'"
+" pour voir le reste du texte.)"
 
 #: src/controller.cpp:1166
 msgid ""
 "It bundles JSON for Modern C++ library, licensed under the MIT License: "
 "https://github.com/nlohmann/json"
-msgstr ""
+msgstr "Il contient la librairie JSON for Modern C++, publiée sous la licence MIT/X Consortium"
```

Comment:
Just to be sure I got the meaning correctly, this means that newsboat contains the json library whose github is linked below right ?

Reply:
I think "bibliothèque" is a best translation than the false friend "librairie".

Reply:
> Just to be sure I got the meaning correctly, this means that newsboat contains the json library whose github is linked below right ?

Yes, that's right.